ta/ta.mk

```diff
@@ -48,28 +48,45 @@ base-prefix := $(sm)-
 
 libname = utils
 libdir = lib/libutils
+ifeq ($(CFG_ULIBS_SHARED),y)
+libuuid = 71855bba-6055-4293-a63f-b0963a737360
+endif
```

Comment:
These are ok but could maybe be centralized in a `lib/uuids.mk` file, a set from a library makefile script.
`ta/ta.mk` looks not the right place.

Reply:
IMO it is the right place for defining the UUID, because it is where we define which user-mode libraries we want to build and include in the TA dev kit, their name, their source directory, and then logically their UUID.
In fact I think the name of the makefile (`ta/ta.mk`) is not very good, especially now that we also have in-tree TAs.

How about I move the `ifeq ($(CFG_ULIBS_SHARED),y)` to `mk/lib.mk` instead of repeating it for each library?